.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: CI
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 21 * * 0'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        java: [8, 11]
+    runs-on: ${{ matrix.os }}
+    env:
+      # define Java options for both official sbt and sbt-extras
+      JAVA_OPTS:   -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+      JVM_OPTS:    -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+    steps:
+      - name: Ignore line ending differences in git
+        if: contains(runner.os, 'windows')
+        shell: bash
+        run: git config --global core.autocrlf false
+
+      - name: Checkout current branch (full)
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java and Scala
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: ${{ matrix.java }}
+
+      - uses: coursier/cache-action@v5
+
+      - name: Setup AWS Credentials
+        shell: bash
+        run: |
+          mkdir ~/.sbt || true && echo "
+          accessKey = test
+          secretKey = test
+          " >> ~/.sbt/.s3credentials_maven.custom
+
+      - name: Build, test, and publishLocal
+        shell: bash
+        run: |
+          sbt -v clean ^test ^publishLocal
+      - name: Scripted Tests
+        # Docker isn't installed on OSX runner - windows tbd  || contains(runner.os, 'windows') }
```

Comment:
the docker dependency hasn't been added yet, but this is here for some documentation.  the existing scripted tests fail on macos / windows on `javax.ws.rs-api` since the `maven.custom` doesn't exist.

```
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	::          UNRESOLVED DEPENDENCIES         ::
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	:: javax.ws.rs#javax.ws.rs-api;2.1: several problems occurred while resolving dependency: javax.ws.rs#javax.ws.rs-api;2.1 {compile=[default(compile)]}:
[info] [warn] 	several problems occurred while resolving dependency: net.java#jvnet-parent;5 {}:
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 5D5CE9BBXV8HEC14; S3 Extended Request ID: xLqJxcR8vB3Lal1Jiyt+RVPKke1udFXxu1Gq6EOrSluXo5PrYGgquppqPzR3kVomiNZL4lU+zTE=; Proxy: null)
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06XEG5FGM1BM043W; S3 Extended Request ID: 6GqVMFkNr3xeVdXfbIviNxGdM9icxZbvHiV9qaV588e/jftdo00yPIkmtU5bZ6z5orIDBuIlZa0=; Proxy: null)
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06XAQZFFK8YJWMX2; S3 Extended Request ID: Ojj6EaDfCM7rGWAPggfe75j6MGKHGNPOVSF4EWpB6MaLejuwa1kQnSu1BMJ6nffegOkOrZxwVqY=; Proxy: null)
[info] [warn] 
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06X4TZ5SJH79BBMR; S3 Extended Request ID: MTKogzztihGupRy24dtKuWwTZwOtx/Mg6SSF7eT8fVGaPcm8zLVQaDbSoSynHlLJUNyDjgQF0tM=; Proxy: null)
[info] [warn] 	several problems occurred while resolving dependency: net.java#jvnet-parent;5 {}:
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06X6FK4WA1VYSWWP; S3 Extended Request ID: R+tj1aG8uVAazSOX5p9+FX04V7VaCoMaogwU/3BEYPPkEVrPbiMuwtZJjKFlpBs2VmNOXzYztBg=; Proxy: null)
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 4SB232HS4KZSANYG; S3 Extended Request ID: 63LhEc/zmYUPghdOYA2XXDiZvvq/z6km4vkI4PAGN0sOWf149Fdur5ly9P2DxK1ihHr+0INwHUU=; Proxy: null)
[info] [warn] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 4SB49219E74NE9FP; S3 Extended Request ID: yyMdPgaSQY910mGNmwVtiqu5n4lFXet/ibxTzjJZ6oy9zEwwGXrxUlT9CXWrWNwcMJV3LJPNMmo=; Proxy: null)
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 
[info] [warn] 	Note: Unresolved dependencies path:
[info] [warn] 		javax.ws.rs:javax.ws.rs-api:2.1 (C:\Users\runneradmin\AppData\Local\Temp\sbt_3c86d309\javax.ws.rs-api\build.sbt#L6-9)
[info] [warn] 		  +- default:javax-ws-rs-api_2.10:0.1-SNAPSHOT
[info] sbt.ResolveException: unresolved dependency: javax.ws.rs#javax.ws.rs-api;2.1: several problems occurred while resolving dependency: javax.ws.rs#javax.ws.rs-api;2.1 {compile=[default(compile)]}:
[info] 	several problems occurred while resolving dependency: net.java#jvnet-parent;5 {}:
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 5D5CE9BBXV8HEC14; S3 Extended Request ID: xLqJxcR8vB3Lal1Jiyt+RVPKke1udFXxu1Gq6EOrSluXo5PrYGgquppqPzR3kVomiNZL4lU+zTE=; Proxy: null)
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06XEG5FGM1BM043W; S3 Extended Request ID: 6GqVMFkNr3xeVdXfbIviNxGdM9icxZbvHiV9qaV588e/jftdo00yPIkmtU5bZ6z5orIDBuIlZa0=; Proxy: null)
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06XAQZFFK8YJWMX2; S3 Extended Request ID: Ojj6EaDfCM7rGWAPggfe75j6MGKHGNPOVSF4EWpB6MaLejuwa1kQnSu1BMJ6nffegOkOrZxwVqY=; Proxy: null)
[info] 
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06X4TZ5SJH79BBMR; S3 Extended Request ID: MTKogzztihGupRy24dtKuWwTZwOtx/Mg6SSF7eT8fVGaPcm8zLVQaDbSoSynHlLJUNyDjgQF0tM=; Proxy: null)
[info] 	several problems occurred while resolving dependency: net.java#jvnet-parent;5 {}:
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 06X6FK4WA1VYSWWP; S3 Extended Request ID: R+tj1aG8uVAazSOX5p9+FX04V7VaCoMaogwU/3BEYPPkEVrPbiMuwtZJjKFlpBs2VmNOXzYztBg=; Proxy: null)
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 4SB232HS4KZSANYG; S3 Extended Request ID: 63LhEc/zmYUPghdOYA2XXDiZvvq/z6km4vkI4PAGN0sOWf149Fdur5ly9P2DxK1ihHr+0INwHUU=; Proxy: null)
[info] 	Forbidden (Service: Amazon S3; Status Code: 403; Error Code: 403 Forbidden; Request ID: 4SB49219E74NE9FP; S3 Extended Request ID: yyMdPgaSQY910mGNmwVtiqu5n4lFXet/ibxTzjJZ6oy9zEwwGXrxUlT9CXWrWNwcMJV3LJPNMmo=; Proxy: null)
[info] 
```

This gets fixed on a future PR using `localstack` to start a local S3 instance for running scripted tests against.

Reply:
@tpunder bump, if you want me to remove this part for now, I can.  But getting CI working again is the first step to get the ball rolling.